ui/src/utils/rounding.ts

```diff
@@ -7,12 +7,17 @@ export enum Precision {
     alignmentLengthMeters,
     measurementMeterDistance,
     measurementKmDistance,
+    curveMeters,
+    angleFractions,
+    distanceMetersInMicrometers,
 }
 
 export function roundToPrecision(n: number, precision: Precision): string {
     switch (precision) {
         case Precision.distanceMeters:
             return n.toFixed(3);
+        case Precision.distanceMetersInMicrometers:
+            return n.toFixed(6);
```

Comment:
Tein nämä `distanceMetersInMicrometers`:it eilen kun luulin että niitä oltaisiin tarvittu, mutta ne näyttäisikin jääneen lopulta turhiksi (varmaan parempi niin, toivottavasti Suomen ratarakentaminen ei ole tämän tarkkuuden juttuja.) Nämä voi siis poistaa.